src/General/Button/LinkButton.tsx

```diff
@@ -6,9 +6,14 @@ import { LinkBtn } from '../../Style/General/ButtonStyle';
 const LinkButton: React.FunctionComponent<Props> = ({
   children,
   className,
+  tag,
   ...defaultProps
 }) => (
-  <LinkBtn className={classNames('aries-linkbtn', className)} {...defaultProps}>
+  <LinkBtn
```

Comment:
I'm wondering if we should allow users to change the tag for this LinkButton in particular. Since it's name specifically states it is a `LinkButton` maybe it should stay as an anchor? Only concern is that it won't be consistent with the rest of the buttons. WDYT?

Reply:
I would like to leave it as it is ^^ If they just want a link, they could have just used a link instead of the button.